.travis.yml

```diff
@@ -12,17 +11,8 @@ cache:
 install: true
 script:
  - 'if [ $TEST_SUITE = "simple" ]; then tasks/e2e-simple.sh; fi'
- - 'if [ $TEST_SUITE = "installs" ]; then tasks/e2e-installs.sh; fi'
```

Comment:
Removed _install_ tests as they are mostly to test `create-react-app` itself, which is stays untouched in this fork, and anyway used the original one from facebook repo.